Update tl721x to Latest Hardware/BLE/Driver Version

damien0x0023 opened the pull request:
- update BLE libs to BLE release/V4.0.4.4
- apply changes in adc/usb due to driver API changes
- enable more buttons in key_pool
- revert default idle for tlx
- add and use rf_zigbee_mode_init for tl721x.

andriy-bilynskyy commented:
The following west manifest projects have changed revision in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/541f19515a7796411ed68352ffd9718472212ca6 (`develop_all_in_one`) | https://github.com/telink-semi/hal_telink/commit/82ac7118d5e966169dd7939580ad51988c2ce1bd (`develop`)| [telink-semi/hal_telink@541f1951..82ac7118](https://github.com/telink-semi/hal_telink/compare/541f19515a7796411ed68352ffd9718472212ca6..82ac7118d5e966169dd7939580ad51988c2ce1bd) |


Additional metadata changed:

| Name | URL | Submodules | West cmds | `module.yml` | 
| ---- | --- | ---------- | --------- | ------------ | 
| hal_telink |  |  |  | ✏ |

⛔ **DNM label due to: 1 project with metadata changes**

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

damien0x0023 committed:
hal: telink: tl721x: udpate BLE libs

- update BLE libs to BLE release/V4.0.4.4
- apply changes in adc/usb due to driver API changes
- enable more buttons in key_pool
- revert default idle for tlx
- add and use rf_zigbee_mode_init for tl721x.

Signed-off-by: Damien Ji <yinghao.ji@telink-semi.com>
